django/test/utils.py

```diff
@@ -50,12 +51,13 @@
     "override_system_checks",
     "tag",
     "requires_tz_support",
-    "setup_databases",
+    "TestDatabaseSetup",
     "setup_test_environment",
     "teardown_test_environment",
 )
 
 TZ_SUPPORT = hasattr(time, "tzset")
+main_db_ready = threading.Event()
```

Comment:
Nit: Looks like this isn't used, so it can probably be removed. Which is great, since mutable global state like this could lead to some strange behaviours.

Reply:
Wops, I didn't notice I created two events, thanks